generic/piranha-tree-sitter/src/test-resources/java/expected_control/XPFlagCleanerInterfaceMethod.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2022 Uber Technologies, Inc.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.piranha;
+
+import dagger.Provides;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.inject.Inject;
+
+class XPFlagCleanerPositiveCases {
+
+  private ExperimentInterface experimentation;
+
+  
+
+  public void conditional_contains_stale_flag() {
+    // BUG: Diagnostic contains: Cleans stale XP flags
+    
+  }
+
+  public void conditional_with_else_contains_stale_flag() {
+    // BUG: Diagnostic contains: Cleans stale XP flags
+    System.out.println("Hi world");
+  }
+
+  public void conditional_with_else_contains_stale_flag_tbool() {
+    // BUG: Diagnostic contains: Cleans stale XP flags
+    
+    System.out.println("Hi world");
+  }
+
+  public void conditional_with_else_contains_stale_flag_tbool(int a) {
+    // BUG: Diagnostic contains: Cleans stale XP flags
+    
+    System.out.println("Hi world");
+  }
+
+  public void conditional_with_else_contains_stale_flag_tbool(int a, bool abc) {
+    // BUG: Diagnostic contains: Cleans stale XP flags
```

Comment:
Let us get rid of BUG: Diagnositcs (a remnant of EP era refactoring)

Reply:
Done!